server/src/api/util/level.js

```diff
@@ -0,0 +1,52 @@
+const { LEVEL_EXP } = require('../constants/levels');
+
+function getLevel(experience) {
+  let index;
+
+  for (index = 0; index < LEVEL_EXP.length; index += 1) {
+    if (LEVEL_EXP[index + 1] > experience) {
+      break;
+    }
+  }
+
+  return Math.min(index, 99);
+}
+
+function getCombatLevel(
+  attackLevel,
+  strengthLevel,
+  defenceLevel,
+  hitpointsLevel,
+  rangedLevel,
+  prayerLevel,
+  magicLevel
+) {
+  // If the player has at least one of the stats as level 0 the calculation becomes incorrect
+  // This is due to the player not being on the Hiscores
+  if (
+    [
+      attackLevel,
+      strengthLevel,
+      defenceLevel,
+      hitpointsLevel,
+      rangedLevel,
+      prayerLevel,
+      magicLevel
+    ].some(level => level === 0)
+  ) {
+    return '-';
+  }
```

Comment:
imo this should just be
```suggestion
  // eslint-disable-next-line prefer-rest-params
  if ([...arguments].some(l => l === 0)) {
    return '-';
  }
```

Reply:
Yep, agree, will do this!